libs/ui-lib/lib/common/components/storage/DisksTable.tsx

```diff
@@ -53,6 +60,80 @@ const SkipFormattingDisk = () => (
   </TextContent>
 );
 
+const getDiskLimitation = (
+  diskName: Disk['name'],
+  hostName: Host['requestedHostname'],
+  holder: Disk,
+) => {
+  if (holder.driveType) {
+    switch (holder.driveType) {
+      case 'LVM':
+        return `LVM logical volumes were found on the installation disk ${
+          diskName as string
+        } selected for host ${hostName as string} and will be deleted during installation.`;
+      case 'RAID':
+        return `The installation disk ${diskName as string} selected for host ${
+          hostName as string
+        }, is part of a software RAID that will be deleted during the installation.`;
+      case 'Multipath':
+        return `The installation disk ${diskName as string} selected for host ${
+          hostName as string
+        } is managed by multipath. We strongly recommend using the multipath device ${
+          holder.name as string
+        } to improve reliability.`;
+    }
+  }
+};
+
+const DiskName = ({
+  disk,
+  disks,
+  host,
+  installationDiskId,
+}: {
+  disk: Disk;
+  disks: Disk[];
+  host: Host;
+  installationDiskId?: string;
+}) => {
+  const isIndented = disk.holders?.split(',').length === 1;
+  let diskLimitations = null;
+
+  if (disk.id === installationDiskId) {
+    const parentDisk = disks.find((d) => disk.holders?.split(',').includes(d.name as string));
+    if (parentDisk) {
+      diskLimitations = getDiskLimitation(disk.name, host.requestedHostname, parentDisk);
+    }
+  }
+
+  return (
+    <>
+      {isIndented && <span style={{ width: '1rem', display: 'inline-block' }} />}
+      {isInDiskSkipFormattingList(host, disk.id) && (
+        <Popover bodyContent={<SkipFormattingDisk />} minWidth="20rem" maxWidth="30rem">
+          <ExclamationTriangleIcon color={warningColor.value} size="sm" />
+        </Popover>
+      )}
+      {'   '}
+      {disk.bootable ? `${disk.name || ''} (bootable)` : disk.name}
+      {diskLimitations && (
+        <>
+          {'    '}
+          <Popover
+            headerContent="Warning"
+            bodyContent={<Alert variant={AlertVariant.warning} isInline title={diskLimitations} />}
+            minWidth="20rem"
+            maxWidth="30rem"
```

Comment:
```suggestion
            minWidth="20rem"
            maxWidth="30rem"
```
Please try using one of the `pf-u-*` classes instead

Reply:
PatternFly provides these props, it should be ok to use them.

Also I think the `pf-u-*` classes are for percentages which isn't really what we need here.